enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -0,0 +1,493 @@
+---
+title: tokenized-auth-enablement-operators-on-cloud
+authors:
+  - "@bentito"
+  - "@jharrington22"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@joelanford" # for OLM, OperatorHub changes
+  - "@bparees" # for OLM, OperatorHub changes
+  - "@csrwng" # for HyperShift changes
+  - "@spadgett" # for Console changes
+approvers:
+  - "@sdodson"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@abutcher" # for CCO API changes, or please suggest alternative
```

Comment:
This should be myself and/or @deads2k

Reply:
Ah, so I wonder if this should be `None`? API proposed changes are just in CCO and possibly removing pod-identity webhook from HyperShift.

Reply:
Ah, nevermind, forgot where `CredentialsRequest` lives. Yep, adding you @JoelSpeed